src/app/modules/common-registration/helpers/registration.helper.ts

```diff
@@ -125,11 +125,35 @@ export function isObservationModelEmptyForRegistrationTid(
   registrationTid: RegistrationTid
 ): boolean {
   if (regModel && registrationTid) {
-    return isEmpty(getRegistationPropertyForModel(regModel, registrationTid));
+    return isEmptyValidation(regModel, registrationTid);
```

Comment:
Kul, generell løsning.
Men jeg tror dette er såpass spesielt at vi heller burde implementere det ved å override isEmpty() for AvalancheObs. Dette fordi vi trenger dette kun når vi registrerer data, og jeg tror de vil være lettere å skjønne unntakene når de ligger der skjemaet er.
Du må gjerne beholde hasAnyDataBesidesPropertyToExclude() i registration.helper, slik at AvalancheObsPage.isEmpty() kan bruke denne.

Reply:
Du bør også sjekke om skred-tidspunktet er det samme som observasjonstidspunktet. Hvis de ikke er like, har brukeren endret tidspunktet selv, og da mener kanskje brukeren at det faktisk er en skredhendelse